Sources/OpenTracing/NoopTracer.swift

```diff
@@ -6,7 +6,10 @@ private struct NoopGlobals {
     static let context = NoopSpanContext()
 }
 
+/// All classes here are no-op and thus are not documented with headerdocs
```

Comment:
Instead, `NoopTracer` should be documented explaining why it exists and how it's useful